index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+  </head>
+  <body>
+
+
+    <header>
+      <img src="./html200-adopt-a-dog/images/adoptadog-logo.svg" alt="Company Logo">
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <a href="./checkout.html">Checkout</a>
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="./dogs.html">Dogs</a>
+        </li>
+        <li>
+          <a href="./blog.html">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main>
+      <section>
+        <h2>Section Header</h2>
+          <article>
+            <h2>Adopt A Dog</h2>
+
+            <img src="./html200-adopt-a-dog/images/home-intro.jpg" alt="A man with a dog">
+
+            <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate
+</p>
+
+          </article>
+
+          <article>
+
+            <img class="dogimg" src="./html200-adopt-a-dogimages/murphy-card.jpg" alt="dog running">
```

Comment:
You shouldn't need the html200-adopt-a-dog part of your src attribute. It should be 

`src="./images/murphy-card.jpg"`